microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -56,119 +52,196 @@ The tutorial steps below walk you through this environment setup.
+The **Overrides** feature lets you store a local copy of the webpage files on the server. When you refresh the page, Microsoft Edge loads the local copy of files instead of the files on the server.
 
-<!-- todo: add section when content is ready (clarify this note) -->
+See also:
+* [Override webpage resources with local copies (Overrides tab)](../javascript/overrides.md)
 
 
 <!-- ====================================================================== -->
-## Create the directory of source files
+## Step 1: Setup
+<!-- https://developer.chrome.com/docs/devtools/workspaces/#setup -->
+
+
+<!-- ------------------------------ -->
+#### Set up the demo
+<!-- https://developer.chrome.com/docs/devtools/workspaces#demo -->
+
+To start this tutorial, in this section, we set up the local directory containing web page source files, and start the localhost test server.
+
+
+<!-- ---------- -->
+###### Clone the MicrosoftEdge / Demos repo to your local drive
+
+To clone the **MicrosoftEdge / Demos** repo to your local drive, including the `/workspaces/` demo folder:
+
+1. Go to [MicrosoftEdge / Demos](https://github.com/MicrosoftEdge/Demos) in a new window or tab.
+
+1. Click the **Code** drop-down button, and then click the **Copy url to clipboard** button.
+
+   The URL is copied to the clipboard: `https://github.com/MicrosoftEdge/Demos.git`
+
+   Or, if you have GitHub Desktop installed, click **Open with GitHub Desktop** to clone the repo, and skip the command prompt step below.
 
-We'll set up the demo files, and then set up DevTools.
+   Or, you can use Visual Studio Code's **Source Control** pane to clone the repo, and skip the command prompt step below.
 
-1. In another window or tab, go to the [Workspaces demo source code](https://github.com/MicrosoftEdge/Demos/tree/main/workspaces).
+1. Assuming you have git installed, open a command prompt.
 
-1. Create an `app` directory, such as `~/Desktop/app` or `C:\Users\myusername\app\`.  Copy `index.html`, `../shared/img/logo.png`, `README.md`, `script.js`, and `styles.css` from the demo source code to your `app` directory.  For the rest of the tutorial, this directory is referred to as `~/Desktop/app` or `C:\Users\myusername\app\`, though you can use a different path.
+1. Clone the repo to your local drive, entering the URL string that you copied from the GitHub repo.  If you use a command prompt:
+
+   ```Shell
+   # example location where the repo directory will be added:
+   cd c:/users/localAccount/github/
+   git clone https://github.com/MicrosoftEdge/Demos.git
+   ```
+
+For details about cloning a repo, see:
+* [Cloning a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) - GitHub docs.
+
+Continue with the next section.
+
+
+<!-- ---------- -->
+###### Create a directory for the localhost server to use
+
+1. Create an `app` directory, such as `~/Desktop/app` or `C:\Users\localAccount\app\`.
```

Comment:
this isn't necessary.  if using the cloned Demos repo, it makes more sense to add the entire cloned /Demos/ dir or the cloned /Demos/demo-to-do/ dir to the Workspace (can switch to a test branch, not "main") instead of needless copying from /Demos/demo-to-do/ dir to a separate /app/ dir.

If we change this approach to skip creating /app/ dir, would need to update text & pngs.
@captainbrosset

Reply:
updated text.  entered fresh Comment to update pngs to not show /app/ dir.